usr/lib/common/lock_sess_mgr.c

```diff
@@ -88,6 +88,12 @@ CK_RV session_mgr_new(CK_ULONG flags, CK_SLOT_ID slot_id,
     so_session = session_mgr_so_session_exists();
     user_session = session_mgr_user_session_exists();
 
+    if (pthread_rwlock_wrlock(&sess_list_rwlock)) {
+        TRACE_ERROR("Read Lock failed.\n");
```

Comment:
Traces message is wrong - this is an rwlock not a read lock.
This is false for all the traces following in this patch.

Reply:
good catch. Fixing this....